### PR TITLE
[Feature][commom] support hadoop-aliyun oss filesystem

### LINKED ISSUE #2864: [feature] hadoop-aliyun oss存储
加入hadoop-aliyun模块，使用oss存储文件资源

### PR DESCRIPTION
**Pull request by quicklyfast**
## What is the purpose of the pull request
support oss filesystem

this closes #2864

This change added tests and can be verified as follows:

*(example:)*
- *edit common.properties*
```
resource.storage.type=OSS
# OSS bucket
fs.oss.bucket=xxxxx
# OSS endpoint
fs.oss.endpoint=http://oss-cn-zhangjiakou.aliyuncs.com
# OSS accessKeyId
fs.oss.accessKeyId=xxxx
# OSS accessKeySecret
fs.oss.accessKeySecret=xxxx
```

### CODE REVIEW

**Comment by CalvinKirs** [comment]
Please add and perfect your unit tests.thx

File: dolphinscheduler-common/pom.xml [hunk 1]
```diff
@@ -313,6 +314,26 @@
...
+				<exclusion>
+					<groupId>org.codehaus.jackson</groupId>
+					<artifactId>jackson-core-asl</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.codehaus.jackson</groupId>
+					<artifactId>jackson-mapper-asl</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
```
Comment (CalvinKirs) [inline dolphinscheduler-common/pom.xml:335]: adding a new jar requires e-mail instructions. You can refer to (https://dolphinscheduler.apache.org/zh-cn/docs/development/DS-License.html) and send an e-mail  to the community after completing the self-check.

**Comment by dailidong** [comment]
does hadoop-aliyun open source? I can't find the related repository url?

**Comment by quicklyfast** [comment]
> does hadoop-aliyun open source? I can't find the related repository url?

hadoop-aliyun is a sub moudle in haddop project, you can visit it on [hadoop repo](https://github.com/apache/hadoop/tree/trunk/hadoop-tools/hadoop-aliyun)

**Comment by dailidong** [comment]
> > does hadoop-aliyun open source? I can't find the related repository url?
> 
> hadoop-aliyun is a sub moudle in haddop project, you can visit it on [hadoop repo](https://github.com/apache/hadoop/tree/trunk/hadoop-tools/hadoop-aliyun)

good，I have another question: does hadoop-aliyun need the version 2.9.2 hadoop jar?  I think 2.7.x is mostly used in production in many companies.

**Comment by quicklyfast** [comment]
> > > does hadoop-aliyun open source? I can't find the related repository url?
> > 
> > 
> > hadoop-aliyun is a sub moudle in haddop project, you can visit it on [hadoop repo](https://github.com/apache/hadoop/tree/trunk/hadoop-tools/hadoop-aliyun)
> 
> good，I have another question: does hadoop-aliyun need the version 2.9.2 hadoop jar? I think 2.7.x is mostly used in production in many companies.

yes, it can't run that use hadoop 2.7.x, because some classes not found

**Comment by dailidong** [comment]
> > > > does hadoop-aliyun open source? I can't find the related repository url?
> > > 
> > > 
> > > hadoop-aliyun is a sub moudle in haddop project, you can visit it on [hadoop repo](https://github.com/apache/hadoop/tree/trunk/hadoop-tools/hadoop-aliyun)
> > 
> > 
> > good，I have another question: does hadoop-aliyun need the version 2.9.2 hadoop jar? I think 2.7.x is mostly used in production in many companies.
> 
> yes, it can't run that use hadoop 2.7.x, because some classes not found

considering hadoop 2.7.x is mostly used, we will merge this PR later.
Thanks for your first contribution, please contact me (wechat: 510570367 or dailidong66@gmail.com), when added, please tell me your github id, thx

**Comment by CalvinKirs** [comment]
no update, thank you for taking the time to complete this task, and look forward to your next contribution. We will support this feature in S3 mode, and I will close this PR.

**Pull request closed without merging**